src/types/FilterTodosBy/FilterTodoBy.ts

```diff
@@ -0,0 +1,5 @@
+export enum FilterTodosBy {
```

Comment:
The file name and the primary export name do not match. Please rename the file to `FilterTodosBy.ts` to match the primary export `FilterTodosBy` 📝